assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/data/useInitialState.ts

```diff
@@ -27,9 +38,17 @@ const useInitialState = ({ ticketId }: BaseProps): StateInitializer => {
 	//sort'em
 	const sortedPrices = sortByPriceOrderIdAsc(unSortedPrices);
 
+	const getPrices = () => {
```

Comment:
It's better to wrap it inside `useCallback`

Reply:
agree, done